tests/integration/popup/popup_core.js

```diff
@@ -158,7 +158,7 @@ asyncTest( "Popup does not go back in history twice when opening on separate pag
 			$( "#go-to-another-page" ).click();
 		},
 		{
-			pagechange: { src: $.mobile.pageContainer, event: "pagechange" + eventNs + "1" }
+			pagechange: { src: $( ".ui-pagecontainer" ), event: "pagechange" + eventNs + "1" }
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?